integrationtests/consts.go

```diff
@@ -0,0 +1,8 @@
+package integrationtests
+
+const (
+	//nolint
```

Comment:
why nolint call here?

Reply:
`gogang-ci` linter detects that these constants are not used, which is false